app/src/lib/stores/app-store.ts

```diff
@@ -438,10 +438,43 @@ export class AppStore extends TypedBaseStore<IAppState> {
     // only emit an update if its changed
     if (currentStep !== this.currentOnboardingTutorialStep) {
       this.currentOnboardingTutorialStep = currentStep
+      this.recordTutorialStepCompleted(currentStep)
       this.emitUpdate()
     }
   }
 
+  private recordTutorialStepCompleted(step: TutorialStep): void {
+    switch (step) {
+      case TutorialStep.PickEditor:
+        this.statsStore.recordTutorialRepoCreated()
```

Comment:
I was initially confused about why this method was called from here. It clicked eventually but I wonder if it would be more obvious and straightforward if we could call `recordTutorialRepoCreated` from `onNewTutorialRepository` rather than relying on this Step to be set.

Reply:
Yup, dropped `recordTutorialRepoCreated` here